### DISCUSSION TITLE
How to download the grapesjs

### DISCUSSION THREAD

**Discussion by nestaho833** [GrapesJS/grapesjs#4340] [General]
Hello everyone,

I am a computer amateur. I found this app in google. The demo looks great! 
But I don't know how to install it. I tried to read the documentation, but it is too difficult for me. :( 
I can't find a one-click installer (like a .exe file)
What should I do?

Thanks
Nesta

**Comment by ebalajier** [comment]
Try this

https://grapesjs.com/docs/getting-started.html
